collections/transactionsCollection.js

```diff
@@ -78,3 +78,93 @@ TransactionsCollection.prototype.leagueFetch = function(leagueKeys, resources, f
 };
 
 // todo: http://fantasysports.yahooapis.com/fantasy/v2/league/{league_key}/transactions;types=waiver,pending_trade;team_key={team_key}
+
+TransactionsCollection.prototype.add_player = function(leagueKey, teamKey, playerKey, cb) {
+  var url = 'http://fantasysports.yahooapis.com/fantasy/v2/league/' + leagueKey + '/transactions?format=json';
+  var xmlData = ' \
```

Comment:
Yes, we need 'POST' request with some XML data to add/drop players. :smiley_cat: 
Refer this site, https://developer.yahoo.com/fantasysports/guide/#post